src/test/app/forms/models/address.ts

```diff
@@ -2,20 +2,38 @@
 /* tslint:disable:no-unused-expression */
 
 import { expect } from 'chai'
-import * as _ from 'lodash'
 import { Validator } from 'class-validator'
 
-import { expectValidationError } from './validationUtils'
-import { Address, ValidationErrors as AddressValidationErrors } from 'forms/models/address'
-import { CorrespondenceAddress, ValidationErrors as CorrespondenceAddressValidationErrors } from 'forms/models/correspondenceAddress'
+import { expectValidationError, generateString, evaluateErrorMsg } from './validationUtils'
+import {
+  Address,
+  ValidationErrors as AddressValidationErrors,
+  ValidationConstants as AddressValidationConstants
+} from 'forms/models/address'
+import {
+  CorrespondenceAddress,
+  ValidationConstants as CorrespondenceAddressValidationConstants,
+  ValidationErrors as CorrespondenceAddressValidationErrors
+} from 'forms/models/correspondenceAddress'
+import { Address as ClaimAddress } from 'claims/models/address'
 
 describe('Address/CorrespondenceAddress', () => {
   [
-    { classFunction: Address, validationErrors: AddressValidationErrors },
-    { classFunction: CorrespondenceAddress, validationErrors: CorrespondenceAddressValidationErrors }
+    {
+      classFunction: Address,
+      validationErrors: AddressValidationErrors,
+      validationConstants: CorrespondenceAddressValidationConstants
+    },
+    {
+      classFunction: CorrespondenceAddress,
+      validationErrors: CorrespondenceAddressValidationErrors,
+      validationConstants: AddressValidationConstants
+    }
   ].forEach((testInput) => {
     const ClassFunction = testInput.classFunction
     const ValidationErrors = testInput.validationErrors
+    const tooLongAddress: number = testInput.validationConstants.ADDRESS_MAX_LENGTH + 1
```

Comment:
These names are confusing to me. When I saw `generateString(tooLongAddress)` and I did not remember the method signature I couldn't know what the value behind `tooLongAddress` might be. For me if we had `addressMaxLength + 1` is quite descriptive unless we can came with good name for `addressMaxLength + 1` like `exceededAddressLength`

Reply:
done